package.json

```diff
@@ -24,11 +24,6 @@
     "mocha": "NODE_ENV='test' mocha \"./tests/*.spec.js\" --watch --require @babel/polyfill --require @babel/register",
     "mocha:unit": "NODE_ENV='test' mocha --watch --require @babel/polyfill --require @babel/register --require ./tests/setup/setupEnzyme.js"
   },
-  "husky": {
```

Comment:
I think we cannot remove this part because before committing changes, it runs lint-staged that makes our code have uniform and consistent format.